.changeset/sixty-taxis-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-css": patch
```

Comment:
You would also need a separate changeset for `itwinui-react` since that is also indirectly affected.

Reply:
Fixed!

Reply:
Great! But you will need to add `itwinui-react` to a *separate* changeset, similar to https://github.com/iTwin/iTwinUI/pull/1909#discussion_r1521684134.

Reply:
Argh, ok. I've separated the changesets into 2.